bigquery/google/cloud/bigquery/magics.py

```diff
@@ -434,6 +444,21 @@ def _cell_magic(line, query):
     else:
         max_results = None
 
+    if not re.search(r"\s", query.rstrip()):
+        table_id = query.rstrip()
```

Comment:
BTW,  stripping whitespace from the same value again is not necessary, we could store the stripped string into a variable the first time we do it.

Reply:
fixed in d663104